### PR TITLE
Make-up

### PR DESCRIPTION
**Pull request by theodorthegreathe**
Cosmetic changes - mainly spaces around emoticons and minor correction of display strings.

### CODE REVIEW

File: src/bot/texts/EN/messages.json [hunk 1]
```diff
@@ -18,24 +18,24 @@
...
   "application_in_progress": "⏳ Your application is still under review.\n\nCurrent status {status_sign}:",
   "application_approved": "🎉 Congratulations! Your application has been approved!\n\nCurrent status {status_sign}:",
-  "application_denied": "😔 Unfortunately, your application was declined ...\n\nCurrent status {status_sign}:",
-  "application_failed": "😥Unfortunately, we couldn't get the status of <b>{app_string}</b> application. Please ensure your application details are correct and try unsubscribing and then subscribing again. If the issue persists, you can reach out to developers.",
-  "current_status": "Current application status is {status_sign}:",
-  "current_status_timestamp": "Current Status {status_sign}:\n\n{status}\n\nLast Updated: <b>{timestamp}</b>",
+  "application_denied": "😔 I regret to tell, your application was declined ...\n\nCurrent status {status_sign}:",
```
Comment (fernflower) [inline src/bot/texts/EN/messages.json:21]: Unfortunately sounds like a better fit imho.

File: src/bot/texts/EN/messages.json [hunk 2]
```diff
@@ -18,24 +18,24 @@
...
   "dialog_cancel": "No worries! You can always start over by using the /subscribe command.",
   "error_invalid_number": "Oops! 🙈 That doesn't seem like a valid application number. Let's try again. Remember, it's all numbers after \"<i>OAM-</i>\" but before \"<i>/ XX-2023</i>\"",
   "error_subscribe": "😔 Sorry, something went wrong while trying to subscribe you. Please give it another shot in a bit. If the issue persists, feel free to contact developers.",
   "error_generic": "😓 Oops! Something didn't go quite right. Please try again later. If the issue keeps popping up, feel free to contact developers.",
   "dialog_completion": "Fantastic! 🌟 You're all set up. I'll keep you posted with updates related to your application!",
   "unknown_command": "🤔 Sorry, I don't know this command.",
-  "unknown_input": "You wanted something?😊 Please tap on /help to get the list of available commands",
-  "unknown_input_funny": "This command is unknown to me, <s>you visaless bag of flesh</s>"
+  "unknown_input": "You wanted something? 😊 Please tap on /help to get the list of available commands",
+  "unknown_input_funny": "Sometimes I do dream of electric sheep..."
```
Comment (fernflower) [inline src/bot/texts/EN/messages.json:40]: Nice :)

File: src/bot/texts/RU/messages.json [hunk 3]
```diff
@@ -19,9 +19,9 @@
   "application_in_progress": "⏳ Ваше заявление всё еще находится в процессе рассмотрения.\n\nТекущий статус {status_sign}:",
   "application_approved": "🎉 Поздравляем! Ваше заявление было одобрено!\n\nТекущий статус {status_sign}:",
   "application_denied": "😔 Нам очень жаль, но похоже ваше заявление было отклонено ...\n\nТекущий статус {status_sign}:",
-  "application_failed": "😥К сожалению нам не удалось получить статус для вашего заявления <b>{app_string}</b>. Пожалуйста, убедитесь что ваши данные верны и попробуйте отписаться, а затем снова подписаться. Если проблема сохраняется, вы можете обратиться к разработчикам.",
-  "current_status": "Текущий статус заявки {status_sign}:",
-  "current_status_timestamp": "Текущий статус {status_sign}:\n\n{status}\n\nПоследняя проверка: <b>{timestamp}</b>",
+  "application_failed": "😥 К сожалению, нам не удалось получить статус для вашего заявления <b>{app_string}</b>. Пожалуйста, убедитесь что ваши данные верны и попробуйте отписаться, а затем снова подписаться. Если проблема сохраняется, вы можете обратиться к разработчикам.",
```
Comment (fernflower) [inline src/bot/texts/RU/messages.json:22]: .. получить статус вашего заявления .. (без для)?

**Review by olegeech-me — CHANGES_REQUESTED** [review]
Since we now have multi-subscription, I increased the limits up to 5 force refresh a day

File: src/bot/texts/EN/messages.json (hunk 1 above)
Comment (theodorthegreathe) [reply src/bot/texts/EN/messages.json]: `"😔 I regret to inform that your application was declined. \n\nCurrent status {status_sign}:",`

**Review by theodorthegreathe — COMMENTED** [review]
All suggestions implemented
I continue learning git and GitHub1

**Review by olegeech-me — APPROVED** [review]

**Pull request merged**